NDTensors/src/Unwrap/src/functions/linearalgebra.jl

```diff
@@ -0,0 +1,11 @@
+function qr(E::Exposed)
+  return qr(unexpose(E))
+end
+
+function eigen(E::Exposed)
+  return eigen(expose(E))
+end
+
+function svd(E::Exposed)
+  return svd(expose(E))
```

Comment:
```suggestion
  return svd(unexpose(E))
```

Reply:
Oh I am still working on decompositions (SVD, eigen QR, etc) so this was an accident that it was added here. Thanks though!